src/app/static/src/app/store/modelCalibrate/actions.ts

```diff
@@ -155,12 +178,38 @@ export const actions: ActionTree<ModelCalibrateState, RootState> & ModelCalibrat
             if (response) {
                 const payload = {
                     data: response.data,
-                    indicatorId: indicatorId
+                    indicatorId
                 } as CalibrateResultWithType
                 commit({type: ModelCalibrateMutation.CalibrateResultFetched, payload: payload});
             }
             commit(`modelOutput/${ModelOutputMutation.SetTabLoading}`, {payload:{tab, loading: false}}, {root: true});
         }
+        updatePlotSelections(commit, payload);
```

Comment:
I agree with Rob that `getResultData` now seems a misleading name for this method since it's committing the selections update as well, so either a rename of at least a comment here explaining why the update plot selections happens here would be good.